### PR TITLE
Someone check

### PR DESCRIPTION
**Pull request by s-oswald**
Something got messed up with my flutter pub get for getting the dependencies. Can someone try to run this on their machine and see if it works? Gotta figure out what the issue is.

### CODE REVIEW

**Review by IcemanOmikron — APPROVED** [review]

**Comment by ggpepp** [comment]
I think it should all work now but test on your systems as well.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # PrepHQ Connect
 
-![](images/PrepHQ%20Final%20Logo.png)
+The PrepHQ Connect app
```
Comment (chwerth) [inline README.md]: Can we keep the image? I don't know if you did this on purpose.

File: lib/views/auth/login.dart [hunk 2]
```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:prephq_connect/views/mentor/mentorscreen.dart';
+import 'package:prephq_connect/views/student/studentscreen.dart';
+import 'package:mongo_dart/mongo_dart.dart' as md;
+
+
+import '../../registrationscreen.dart';
+
+class MyHomePage extends StatefulWidget {
```
Comment (chwerth) [inline lib/views/auth/login.dart]: Can we name this `LoginScreen` or `LoginPage`? Makes it a little easier to immediately grasp what the widget is

File: lib/views/student/studentscreen.dart [hunk 3]
```diff
@@ -29,9 +30,15 @@ class _StudentScreenState extends State<StudentScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
-        child: _widgetOptions.elementAt(_selectedIndex),
+      body: IndexedStack(
```
Comment (chwerth) [inline lib/views/student/studentscreen.dart:33]: Out of curiosity, what makes `IndexedStack` a better choice here?

**Review by chwerth — CHANGES_REQUESTED** [review]
This is a massive amount of progress, really great job @s-oswald.

My last random comment is, with the new routing you implemented the android back button is now willing to take you back to the Login Screen if you hit it right after you login. Is there a simple way to pop the login screen off the stack once we login?

Also, since the restructuring was so major, it looks like we need to resolve a few conflicts

**Review by chwerth — APPROVED** [review]

**Comment by chwerth** [comment]
@s-oswald I think I made all the changes I needed to, feel free to merge whenever

**Pull request merged**